src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -423,6 +423,47 @@ protected function get_url_date_format() {
 		return 'Y-m';
 	}
 
+	/**
+	 * Creates a HTML link and "fast forward" message to append to the "no events found" message.
+	 *
+	 * @param bool  $canonical         Whether to return the canonical (pretty) version of the URL or not.
+	 * @param array $passthru_vars     An optional array of query variables that should pass thru the method untouched
+	 *                                 in key and value.
+	 * @return string                  The html link and message.
+	 */
+	public function fast_forward( $canonical = false, array $passthru_vars = [] ) {
+		if ( ! empty( $this->context->get( 'tribe-bar-date' ) ) ) {
+			$date = $this->context->get( 'tribe-bar-date' );
+		} else if ( ! empty( $this->context->get( 'event_date' ) ) ) {
```

Comment:
:warning: Usage of ELSE IF is discouraged; use ELSEIF instead

<sub>hash: dc7ca46e502d3272df17a44b9703ed9e</sub>